modules/local/changeo/changeo_parsedb_select.nf

```diff
@@ -22,7 +22,7 @@ process CHANGEO_PARSEDB_SELECT {
     def args2 = task.ext.args2 ?: ''
     if (meta.locus == 'IG'){
```

Comment:
Is this case sensitive? Could it happen that locus is lower case, and then we should check  'IG' and 'ig'?

Reply:
I will correct it in the follow-up PR where there is also a change related to this, thanks!